src/scrapers/max.ts

```diff
@@ -258,8 +258,8 @@ class MaxScraper extends BaseScraperWithBrowser {
       fields: createLoginFields(credentials),
       submitButtonSelector: '#login-password #send-code',
       preAction: async () => {
-        if (await elementPresentOnPage(this.page, '#closePopup')) {
-          await clickButton(this.page, '#closePopup');
+        if (await elementPresentOnPage(this.page, '.close')) {
```

Comment:
I ran it locally and for me it manages to scrape with a popup that appears. But they might have various popup types.

If you are sure this is needed, you can keep both logics, they doesn't interrupt each other. so you will have something like 

```

        if (await elementPresentOnPage(this.page, '#closePopup')) {      
           await clickButton(this.page, '#closePopup');          
        }
              
        if (await elementPresentOnPage(this.page, '.close')) {      
           await clickButton(this.page, '.close');          
        }
```